helix-core/src/main/java/org/apache/helix/manager/zk/ZkBaseDataAccessor.java

```diff
@@ -85,13 +88,20 @@ public AccessResult() {
 
   private final HelixZkClient _zkClient;
 
+  //TODO: to be deprecated for client usage
   public ZkBaseDataAccessor(HelixZkClient zkClient) {
     if (zkClient == null) {
       throw new NullPointerException("zkclient is null");
     }
     _zkClient = zkClient;
   }
 
+  public ZkBaseDataAccessor(String zkAddress, ZkSerializer zkSerializer) {
```

Comment:
Should we provide` public ZkBaseDataAccessor(String zkAddress)` as well? This would be the default constructor using ZnRecordSerializer when a custom serializer is not needed.

Reply:
done.